libs/blocks/text/text.js

```diff
@@ -76,7 +76,21 @@ function decorateLinkFarms(el) {
   });
 }
 
-export default function init(el) {
+export async function loadMnemonicList(foreground) {
+  try {
+    const { base } = getConfig();
+    const stylePromise = new Promise((resolve) => {
+      loadStyle(`${base}/blocks/mnemonic-list/mnemonic-list.css`, resolve);
+    });
+    const loadModule = import('../mnemonic-list/mnemonic-list.js')
+      .then(({ decorateMnemonicList }) => decorateMnemonicList(foreground));
+    await Promise.all([stylePromise, loadModule]);
+  } catch (err) {
+    window.lana?.log(`Failed to load mnemonic list module: ${err}`);
```

Comment:
@robert-bogos might have some suggestions on how to categorize this.

Reply:
Tags are required to LANA logs in order to facilitate filtering them in Splunk. For this particular use case I would go for:
```suggestion
    window.lana?.log(`Failed to load mnemonic list module: ${err}`, { tags: 'errorType=error,module=text' });
```
You can see examples and read more about it in [here](https://github.com/adobecom/milo/pull/1463).